cron/internal/data/gitlab-projects-selected.csv

```diff
@@ -0,0 +1,100 @@
+https://gitlab.com/gitlab-org/gitlab-foss,
```

Comment:
while we support not having the csv header, lets keep it here for consistency with `projects.csv`